0800-appendix-198236-trib-to-kitwanga.Rmd

```diff
@@ -25,7 +25,7 @@ my_site <- 198236
 PSCIS crossing `r as.character(my_site)` is located on
 `r fpr::fpr_my_pscis_info()`, on the `r fpr_my_pscis_info(col_pull = road_name)`. This crossing was located
 `r round(fpr::fpr_my_bcfishpass(col_pull = downstream_route_measure)*0.001, 1)`km
-upstream from the confluence with the Kitwanga River. This river drains into Kitwanga Lake (a.k.a Kitwancool Lake or Gitanyow Lake). The underlying territory of this area belongs to the Gitanyow, an Indian reserve community of the Gitxsan first nations.  
+upstream from the confluence with the Kitwanga River. This river drains into Kitwanga Lake (a.k.a Kitwancool Lake or Gitanyow Lake). The underlying territory of this area belongs to the Gitanyow Nation, a First Nations community of the Gitxsan.  
```

Comment:
did a bunch of work to provide context and references to reconcile what could be considered disrespectful and basically a repeat of what happened regarding Sik-E-Dakh memo.  This effort appears to have been in vain and a missed opportunity to show diligence and respect.